tests/gold_tests/logging/all_headers_sanitizer.sh

```diff
@@ -21,5 +21,5 @@ do
     # individual headers on separate lines, then sort the lines.
     #
     echo $LN | sed 's/}}/}}\
-/g' | sort
+/g' | sort --ignore-nonprinting  --dictionary-order
```

Comment:
I didn't run the tests on all machines, but I've verified that this sort command works on macOS, FreeBSD, and various Linux flavors.